src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -2889,13 +2871,10 @@ export default class EthereumApi implements Api {
       throw new Error("from not found; is required");
     }
 
-    const fromString = tx.from.toString();
+    const fromAddress = new Address(tx.from.toString());
```

Comment:
I bet we can change `BaseTransaction`'s `from` type to `Address` and then we don't need to do the `toString()` dance here.